main.go

```diff
@@ -105,6 +106,10 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := validation.InitOutOfServiceTaintSupportedFlag(mgr.GetConfig()); err != nil {
+		setupLog.Error(err, "unable to verify out-of-service taint support. out-of-service taint isn't supported")
```

Comment:
I think this message is a little bit misleading.
We receive an error when we are unable to verify the K8s version which can indicate whether the out-of-service taint is supported.

```suggestion
		setupLog.Error(err, "Unable to verify Kubernetes version for indicating the out-of-service taint support. out-of-service taint isn't supported")
```

Reply:
Fixed it.